src/ccpp_fields_idx.h

```diff
@@ -29,7 +29,7 @@ extern "C"
 #endif
 
 
-#define  CCPP_FIELD_IDX_MAX     500
```

Comment:
Note that this change was necessary after the PBL_generic interstitial update, because the number of fields for FV3 exceeded 500.